### PR TITLE
Provide a way to get expose url from tree view

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #404

### CODE REVIEW

File: src/tekton/addtrigger.ts [hunk 1]
```diff
@@ -100,7 +100,11 @@ export async function k8sCreate(trigger: TriggerTemplateKind | EventListenerKind
     vscode.window.showErrorMessage(`Fail to deploy Resources: ${getStderrString(result.error)}`);
     return false;
   }
-  if (trigger.kind === RouteModel.kind && !result.error) vscode.window.showInformationMessage('Trigger successfully created.');
+  if (trigger.kind === RouteModel.kind && !result.error) {
+    const routeInfo = await cli.execute(Command.getRoute(trigger.metadata.name));
+    const routeHost = JSON.parse(routeInfo.stdout).spec.host;
+    vscode.window.showInformationMessage(`Trigger successfully created. Expose URL: http://${routeHost}`);
```
Comment (evidolob) [inline src/tekton/addtrigger.ts:106]: Why only `http` ? What about `https`?

File: src/tekton/triggertemplate.ts [hunk 2]
```diff
@@ -3,14 +3,48 @@
...
+    const listEventListener = JSON.parse(result.stdout).items;
+    if (listEventListener.length === 0) {
+      vscode.window.showInformationMessage('Expose URl not available');
+      return null;
+    }
+    for (const eventListener of listEventListener) {
+      for (const triggers of eventListener.spec.triggers) {
+        if (triggers?.template?.name === trigger.getName()) {
+          await TriggerTemplate.getExposeURl(eventListener.status.configuration.generatedName);
+          vscode.window.showInformationMessage('Expose URl successfully copy');
```
Comment (evidolob) [inline src/tekton/triggertemplate.ts:29]: Maybe `Expose URl successfully copied` ? @siamaksade ?

File: src/tekton/triggertemplate.ts [hunk 3]
```diff
@@ -3,14 +3,48 @@
...
+        }
+      }
+    }
+    vscode.window.showInformationMessage('Expose URl not available');
+  }
+
+  static async getExposeURl(name: string): Promise<void> {
+    const result = await tkn.execute(Command.getRoute(name));
+    const exposeURL = JSON.parse(result.stdout).spec.host;
+    vscode.env.clipboard.writeText(`http://${exposeURL}`);
```
Comment (evidolob) [inline src/tekton/triggertemplate.ts:48]: Same question about http protocol.

File: src/tekton/addtrigger.ts (hunk 1 above)
Comment (sudhirverma) [reply src/tekton/addtrigger.ts:106]: for the secure route, we can see in TLS termination https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/routes.html#secured-routes I will update the code

**Review by evidolob — APPROVED** [review]

**Pull request merged**